versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.63.0"
+      version = ">= 3.63.0, < 4.0.0"

```

Comment:
4.0.0 split stuff out of existing resources such as splitting the retention policy out of s3 buckets. If we ever want to upgrade, this will require a task.